src/test/java/bio/terra/service/snapshot/SnapshotIntegrationTest.java

```diff
@@ -276,4 +281,50 @@ private SnapshotRequestModel snapshotByQueryRequestModel(DatasetModel dataset) t
                 + ".sample.id ='sample6'");
     return requestModel;
   }
+
+  @Test
+  public void testCreateSnapshotWithPolicies() throws Exception {
+    DatasetModel dataset = dataRepoFixtures.getDataset(steward(), datasetId);
```

Comment:
Yeah, I see how it'd be nice to be able to do this without having to create an entire new dataset :(